src/_constants/ActionTypes.js

```diff
@@ -53,3 +53,7 @@ export const UPDATE_VIDEO_LIST = 'UPDATE_VIDEO_LIST';
 
 export const WATCHLIST_FAVOR_ASSET = 'WATCHLIST_FAVOR_ASSET';
 export const WATCHLIST_UNFAVOR_ASSET = 'WATCHLIST_UNFAVOR_ASSET';
+
+export const ADD_TOKEN = 'ADD_TOKEN';
```

Comment:
I thought about this, I thought that this could be clearer from the point of caller, not sure if there's best practice for this as it's too use-case specific ~ 

I'll search and reply =)